packages/server/lib/browsers/memory/cgroup-v1.ts

```diff
@@ -0,0 +1,47 @@
+import { exec } from 'child_process'
+import util from 'util'
+
+const execAsync = util.promisify(exec)
+
+const getTotalMemoryLimit = async () => {
```

Comment:
For these new 'memory' files, could you go through and add code comments and js doc for the functions? I'm sure theres context currently in your head that would help us manage and understand these methods in the future.

Reply:
added documentation: [5507f19](https://github.com/cypress-io/cypress/pull/25462/commits/5507f19535d97bd0e67db3a69ce6688b62f11eef)